answer/naka1998/JS/26/tanuki.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <textarea name="" id="form" cols="70" rows="10"></textarea>
+    <button onclick="tanuki()">たぬき</button>
+    <div id="result"></div>
+    <script>
+        function tanuki(){
+            let textIn = document.getElementById("form").value;
+            let textOut = textIn.replace(/たた+/g,"た");
```

Comment:
','のあとはスペース入れましょう
e.g. 
```suggestion
            let textOut = textIn.replace(/た+/g, "た");
```

また、正規表現 `/たた+/`でも大丈夫ですが `/た+/`でも動く気がしています。今一度確認してみてください。
正規表現における `+` の使い方ですね